Tests/Opc.Ua.Core.Tests/Types/Utils/UtilTests.cs

```diff
@@ -87,6 +87,68 @@ public void ToHexBigEndian()
             var hexutil = Utils.ToHexString(blob, true);
             Assert.AreEqual(hex, hexutil);
         }
+
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [Test]
+        public void RelativePathParseNumericStringNonDeep()
+        {
+            TypeTable typeTable = new TypeTable(new NamespaceTable());
+            string str = "/11";
+            Assert.AreEqual(str,
+                RelativePath.Parse(str, typeTable).Format(typeTable));
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [Test]
+        public void RelativePathParseNumericStringDeepPath()
+        {
+            TypeTable typeTable = new TypeTable(new NamespaceTable());
+            string str = "/123/789";
+            Assert.AreEqual(str,
+                RelativePath.Parse(str, typeTable).Format(typeTable));
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [Test]
+        public void RelativePathParseAlphanumericStringPath()
+        {
+            string str = "/123A/78B9";
+            // In Little Endian it's written as CE FA
+            Assert.AreEqual(str, RelativePath.Parse(str, new TypeTable(new NamespaceTable())).Format(new TypeTable(new NamespaceTable())));
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [Test]
+        public void RelativePathParseAlphanumericStringPath2()
+        {
+            string str = "/AA123A/bb78B9";
+            // In Little Endian it's written as CE FA
+            Assert.AreEqual(str, RelativePath.Parse(str, new TypeTable(new NamespaceTable())).Format(new TypeTable(new NamespaceTable())));
+        }
+
+        [Test]
+        public void RelativePathParseAlphaStringPath()
```

Comment:
Please add the summary fields for the tests

Reply:
okay (done)